practices/documentation_testing.md

```diff
@@ -0,0 +1,106 @@
+> Tracking why people visit you is crucial to understand which issues and questions bring people to you. Knowing what brings people to you, you can tailor your docs to provide a better experience to your visitors.
+
+#### Clicks and download tracking
+Consider logging how many people download your code samples. For this logging, you can use software like [Google Tag Manager](http://tagmanager.google.com). With Google Tag Manager, you can use Custom events, for example scroll rate or download rate, to gain better insight on your Docs performance.
+> Track any action that is important for you. From Sing-ups to finishing demo, this information directly makes difference in improving developer experience.
+
+### Implementing Active Testing
+Following Active Testing methods can give you insight on your Documentation DX.
+#### Questionnaires
+Questionnaires give your readers a way to directly give their feedback. You can gather interesting information through it. Moreover, if you reach out after the questionnaire you can improve community relationships.
+> Forms and questionnaires are powerful tool if used correctly. With a good form, you can build rapport and turn this rapport into better dx.
+
+#### In-house Testing
+One of the best things you can always do is to test your documentation In-house. Read through your docs. Let your developers try the demos and gather their thoughts. Take people with different background and skillset. 5 to 6 people is enough to start improving your developer experience.
+> Anyone around you can help you with in-house testing. Test if you can understand your docs. Test if your sample codes work. Test what happens if you auto-translate your docs. Be emphatic. Use my favorite definition: _Empathy is connecting with emotions, not with experience._
+
+#### Outsourced Testing
+If you don’t want to spend your resources on In-house Testing, you can try outsourcing it. Services like [Betatesting](https://betatesting.com/beta-testers), [Usertesting](https://www.usertesting.com), or [Devskiller](https://devskiller.com) can sometimes connect you with good testing developer. You can connect with developer community at places like [Dev](https://dev.to) or [DXheroes Slack](https://join.slack.com/t/developerexperiencehq/shared_invite/enQtNzU5NjUyNDk2MDM1LTRmNjM5ZTllOTc1NmU5ZWRhNGNkODU1ZmY0ZWExYWUyN2YyNzM4ZTMzNjIyNWJhNTRmNjA5ZTVhMGRmZjM4ZjY). Alternatively, you can hire freelancers through platforms like [Freelancer](http://freelancer.com), [Upwork](https://www.upwork.com), [Fivver](https://www.fiverr.com) or [Peopleperhour](https://www.peopleperhour.com).
+> You can always outsource the testings if you don’t want to use in-house testing or you want additional data. Outsourcing your testing is not as expensive as you may think, having around 5 - 6 testers is generally enough to understand the dx you provide.
+
+## Resources for Documentation Testing
```

Comment:
The paragraph with Common Pitfalls is missing?

Reply:
The article is lengthy and I've wanted to shorten it. However, you are right. Pitfalls will be added ASAP